rdoc_alfred_workflow.Rmd

```diff
@@ -0,0 +1,52 @@
+```{r, include=FALSE}
```

Comment:
The first line should be an '# <space> <Title>'. The second line should be blank and the third line should be your name

Reply:
Hi sorry for the late reply, I missed the notification on this pull request. I've made the changes and committed them. Do I need to submit a new pull request or will this one be merged?